nixos/modules/installer/netboot/netboot.nix

```diff
@@ -18,6 +18,10 @@ with lib;
 
   };
 
+  imports = [
+     (mkRemovedOptionModule [ "system" "build" "netbootRamdisk" ] "`system.build.netbootRamdisk` was a special initrd for netbooting, it is not necessary to use it as `system.build.initialRamdisk` contains everything you need now and is standard across nixpkgs.")
```

Comment:
```
% nix-build -A netboot.x86_64-linux nixos/release.nix
error: evaluation aborted with the following error message: 'cannot find attribute `system.build.netbootRamdisk''
```

This happens because `system.build` in an option, so you can't nest another option under it like this.

Maybe the thing to do is just something like this? I haven't tested:

```nix
  system.build.netbootRamdisk = lib.warn "…";
```

Reply:
Ah! Alright, it makes sense.